### PR TITLE
More js wrapper tests

### PR DESCRIPTION
**Pull request by aarthykc**
As a follow up to #25 and #22 this PR adds more tests for the javascript wrappers for the GridStore and GridStoreBuilder. The functions we should add tests for are: 

1. insert()
2. finish()
3. append() 
4. reader

cc @apendleton, @Nmargolis

### CODE REVIEW

File: test/bindings/test.js [hunk 1]
```diff
@@ -3,25 +3,52 @@
...
-    store.insert(id, entries);
-    t.ok(store);
+    t.throws(() => store.insert(), 'not enough arguments');
+    store.insert({ phrase_id: 0, lang_set: [0] }, [{ id: 0, x: 0, y: 0, relev: 0.5, score: 2, source_phrase_hash: 0 }]);
+    store.insert({ phrase_id: 1, lang_set: [0, 1, 2, 3] }, [{ id: 2, x: 2, y: 2, relev: 0.6, score: 3, source_phrase_hash: 0 }]);
+    store.finish();
+
+    const reader = new addon.GridStore(tmpDir.name);
+    t.deepEquals(reader.get({ phrase_id: 0, lang_set: [0] }), [ { relev: 1, score: 2, x: 0, y: 0, id: 0, source_phrase_hash: 0 } ], 'able to find the correct gridEntry inserted by insert()');
+    t.deepEquals(reader.get({ phrase_id: 1, lang_set: [0, 1, 2, 3] }), [ { relev: 0.6000000238418579, score: 3, x: 2, y: 2, id: 2, source_phrase_hash: 0 } ], 'able to find the correct gridEntry inserted by insert()');
```
Comment (aarthykc) [inline test/bindings/test.js]: per conversation with @apendleton, 
> relev:`0.6000000238418579` 
should be fixed when the next PR that he's working on lands

Comment (aarthykc) [reply test/bindings/test.js]: Fixed via #27

File: test/bindings/test.js [hunk 2]
```diff
@@ -3,25 +3,72 @@
...
     const store = new addon.GridStoreBuilder(tmpDir.name);
     t.ok(store);
+    rimraf(tmpDir.name);
     t.end();
 });
 
-tape('JsGridStoreBuilder insert', (t) => {
+tape('GridStoreBuilder insert()', (t) => {
     const tmpDir = tmp.dirSync();
     const store = new addon.GridStoreBuilder(tmpDir.name);
```
Comment (apendleton) [inline test/bindings/test.js]: nitpick: `store` as the name of the builder, when the type of the reader is `GridStore`, is surprising. Caught me off guard in the next section when you were trying to insert into a store. writer/reader, builder/reader, builder/store -- however you want to do it is fine, but this particular combination is a little confusing.

Comment (aarthykc) [reply test/bindings/test.js]: good point! changed it to builder/reader

**Pull request merged**